### ISSUE TITLE
Windows 10 click count

### ISSUE THREAD

**Issue by DigitalMystery** [phfsantos/cordova-plugin-smaato#13]
I finally got this set up in my cordova project and do receive the banner ads.  Also, when I click on an ad the ad does open up into the window like I would expect however my click count does not change in Smaato even after waiting a day.  Has anybody seen this and if so is there a config or something I am missing?  Currently this is all I have:

>  var simpleAd = new Smaato(newEle, {
        publisherId: myPubId,
        adId: myAdId
    });
    $timeout(function () {
        simpleAd.show();
    }, 7000);

**Comment by sergip76** [comment]
Hi DigitalMystersy, I have integrated the plugin like you, but I don't receive any ad. Always an error. I have just created my account. Do you know if I have to wait for any validation.
I have tried it in Windows and Android platforms but I receive always an error and no ads.

**Comment by DigitalMystery** [comment]
I only attempted it on Windows because I use admob for android and iOS.  I do not remember if there was a wait....  but if anything just give it a day and see if they start coming thru.  I will say I never really got them working correctly.  The click count always stayed at 0 and Smaato's help was not existent.  I reached out to them to see if there was anything they could see on their end and never got a response.  I took it out and going to try the windows ad service.  I am in the process of updating my application to visual studio 2017 and going to try to use their Microsoft Advertising SDK when I have time.  If you ever get it working please let me know!  I might have a question for you ;-)

**Comment by sergip76** [comment]
I would try also Microsoft Advertising SDK with Visual Studio but it's only for UW Apps and not for Cordova apps. I haven't found any plugin to implement Microsoft Advertising SDK.

**Comment by DigitalMystery** [comment]
Oh...  Well on the page I was looking at it made it sound like you could.  Here is the bullet point I read, "Show banner ads in your HTML/JavaScript app."
This lead me to believe its possible.  I played around real quick in my 2017 environment and it looked like I could add the reference to the SDK.  I may not be able to and if so then I will look for another path after that.  Here is the link I "quickly" read about their SDK.
https://docs.microsoft.com/en-us/windows/uwp/monetize/adcontrol-in-html-5-and-javascript

**Comment by sergip76** [comment]
Yes, I had already read this page, but it explains how to use Microsoft Advertising SDK as extension for UWP app, not for Cordova. Cordova can't work with UWP extensions to use native code, only with plugins.

It seems that the only way to use ads of inapp purchases with Windows 10 is to create a native UWP app. I can't understand how Microsoft is trying to increase the amount the apps on his store without doing any effort to improve multiplatform apps. I think it's a waste of time doing apps only for Windows 10.

**Comment by DigitalMystery** [comment]
gotcha..... and agreed..... please let me know if you get something to work!

**Comment by sergip76** [comment]
Sure! I will do it.
